scripts/variantstore/wdl/GvsExtractCohortFromSampleNames.wdl

```diff
@@ -13,19 +13,20 @@ workflow GvsExtractCohortFromSampleNames {
     Array[String]? cohort_sample_names_array
     File? cohort_sample_names
 
-    String query_project
     String gvs_project
     String gvs_dataset
     String call_set_identifier
-    String cohort_table_prefix
+    String cohort_table_prefix = call_set_identifier
+    String query_project = gvs_project
 
     # not using the defaults in GvsPrepareCallset because we're using pre created datasets defined by the caller
```

Comment:
tbh I don't understand this comment but I'm wondering if it's still "correct" with these changes?